docs/configuration.md

```diff
@@ -1233,13 +1260,15 @@ Apart from these, the following properties are also available, and may be useful
   <td>
     How many finished executions the Spark UI and status APIs remember before garbage collecting.
   </td>
+  <td>1.5.0</td>
```

Comment:
SPARK-8861 and SPARK-8862, commit ID: ebc3aad272b91cf58e2e1b4aa92b49b8a947a045#diff-81764e4d52817f83bdd5336ef1226bd9